src/storage/index/IndexPolicyMaker.h

```diff
@@ -0,0 +1,117 @@
+/* Copyright (c) 2020 vesoft inc. All rights reserved.
+ *
+ * This source code is licensed under Apache 2.0 License,
+ * attached with Common Clause Condition 1.0, found in the LICENSES directory.
+ */
+
+#ifndef STORAGE_INDEXPOLICYMAKER_H
+#define STORAGE_INDEXPOLICYMAKER_H
+#include "base/Base.h"
+#include "meta/SchemaManager.h"
+#include "storage/CommonUtils.h"
+#include "storage/index/IndexOptimizer.h"
+
+namespace nebula {
+namespace storage {
+
+enum class PolicyType : uint8_t {
```

Comment:
I am confused about this enum.  Why we need it.

Reply:
When we traverse the expression tree, we terminate the traversal when  `OR` Operator occur.  if so , the  `PolicyType = SIMPLE_POLICY`
Also, when create scan policy, if the `PolicyType = SIMPLE_POLICY` ,  The logic of policy creation will be skipped。

Reply:
resolved.